src/main/kotlin/com/hj/leetcode/kotlin/problem808/Solution.kt

```diff
@@ -0,0 +1,52 @@
+package com.hj.leetcode.kotlin.problem808
+
+/**
+ * LeetCode page: [808. Soup Servings](https://leetcode.com/problems/soup-servings/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(1) and Space O(1) since the probability will reach 1.0-tolerance when n is
+     * greater than certain value n0;
+     */
+    fun soupServings(n: Int): Double {
+        val memoization = hashMapOf<RemainingSoup, Double>()
+        val tolerance = 1e-5
+        val unitServeMl = 25
```

Comment:
the meaning of `unitServeMl` is not clear

Reply:
Resolved at https://github.com/hj-core/hj-leetcode-kotlin/pull/114/commits/e44919d4a443ef040c56064e68d0e1b44e13149b